src/Illuminate/Database/Query/Builder.php

```diff
@@ -1562,7 +1564,7 @@ public function exists()
 	 * Retrieve the "count" result of the query.
 	 *
 	 * @param  string  $columns
-	 * @return int
+	 * @return mixed
```

Comment:
@codelight89 We should avoid mixed if we can, because it's pretty useless documentation.